python/mantid_converters.py

```diff
@@ -0,0 +1,34 @@
+import dataset as ds
+from mantid.simpleapi import *
+
+def ConvertWorkspase2DToDataset(ws):
+    d = ds.Dataset()
+    if ws.getNumberHistograms() == 0:
+        return d
+    if ws.isCommonBins():
+        d[ds.Coord.Tof] = ([ds.Dim.Tof], ws.readX(0))        
+    else:
+        d[ds.Coord.Tof] = ([ds.Dim.Spectrum, ds.Dim.Tof], (ws.getNumberHistograms(), len(ws.readX(0))))
+        for i in range(ws.getNumberHistograms()):
+            d[ds.Coord.Tof][ds.Dim.Spectrum, i] = ws.readX(i);
+        
+    
+    d[ds.Data.Value] = ([ds.Dim.Spectrum, ds.Dim.Tof], (ws.getNumberHistograms(), len(ws.readY(0))))     
+    d[ds.Data.Variance] = ([ds.Dim.Spectrum, ds.Dim.Tof], (ws.getNumberHistograms(), len(ws.readE(0))))
+    for i in range(ws.getNumberHistograms()):
+        d[ds.Data.Value][ds.Dim.Spectrum, i] = ws.readY(i)
+        d[ds.Data.Variance][ds.Dim.Spectrum, i] = ws.readE()
+    return d
+
+def ConvertEventWorkspaseToDataset(ws):
+    d = ds.Dataset()
+    nHist = ws.getNumberHistograms()
+    d[ds.Coord.Position] = ([ds.Dim.Position], (nHist,))        
+    d[ds.Data.Events] = ([ds.Dim.Position], (nHist,))    
+   
+    for i in range(nHist):
```

Comment:
You are not adding `e` to `d`? Maybe use `for i, e in enumerate(d[ds.Data.Events].data):`